lib/utils/yamlAstParser.js

```diff
@@ -24,6 +25,11 @@ const parseAST = (ymlAstContent, astObject) => {
   let newAstObject = astObject || {};
   if (ymlAstContent.mappings && _.isArray(ymlAstContent.mappings)) {
     _.forEach(ymlAstContent.mappings, (v) => {
+      if (!v.value) {
+        console.log(`Serverless: ${chalk.red(`Your serverless.yml has an invalid value with key: ${v.key.value}. Ignoring...`)}`);
```

Comment:
You might want to use `BbPromise.reject`. console.log can't use the Framework since following the lint rule

Reply:
In addition, you have faced `Line 29 exceeds the maximum line length of 100` lint error.
You need to spirit the line something like this
```javascript
const errorMessage = [
 `${chalk.red(`Your serverless.yml has an invalid value with key: `,
 `${v.key.value}. Ignoring...`,
 ].join('');
```